src/btree/bt_vrfy.c

```diff
@@ -281,12 +281,8 @@ __verify_key_hs(WT_SESSION_IMPL *session, WT_ITEM *key, WT_CELL_UNPACK *unpack,
 
     /*
      * Open a history store cursor positioned at the end of the data store key (the newest record)
-     * and iterate backwards until we reach a different key or btree. We need to do this whilst
-     * holding a snapshot to ensure that modifications done whilst iterating the history store don't
-     * cause false positives.
+     * and iterate backwards until we reach a different key or btree.
      */
-    __wt_txn_get_snapshot(session);
```

Comment:
How come we don't need this anymore?

Reply:
We prematurely decided to hold a snapshot to prevent history store iterations from changes in parallel. Here is why getting a snapshot wont work:
1. cursor->search() refreshes a snapshot and we do search() as part of going over history.
2. History store doesnt follow normal transaction semantics, since txn-ids are from the original transaction, so even if we were to hold a snapshot, I don't think it would work as expected.

There is a ticket to address these issues: WT-5606, when we get the time to.